sympy/solvers/solveset.py

```diff
@@ -871,3 +872,162 @@ def solveset(f, symbol=None):
             return solveset_real(f, symbol)
         else:
             return solveset_complex(f, symbol)
+
+
+###############################################################################
+################################ LINSOLVE #####################################
+###############################################################################
+
+
+def eq_to_matrix(equations, *symbols):
```

Comment:
Matlab has some of the worst variable names of all programming languages. Definitely don't follow their lead on that.